src/api/keyword/asyncGetPostCountList.js

```diff
@@ -0,0 +1,15 @@
+import fetchHandler from "..";
+import { BASE_URL, PERIOD_TYPE } from "../../config/constants";
+
+const asyncGetPostCountList = async (keywordId, cursorId = "", unit = PERIOD_TYPE.WEEKLY) => {
+  const fetchInfo = {
+    url: `${BASE_URL}/posts/${keywordId}/postCount`,
+    params: `?unit=${unit}&cursorId=${cursorId}`,
+  };
```

Comment:
저희 unit을 주간으로만 하기로 결정했기에, API 명세서에서 unit을 삭제한대로 위의 코드에서도 unit은 삭제해도 좋을 것 같습니다. 놔두셔도 좋다고는 생각하지만, 통일성을 위해 의견드립니다.

```suggestion
const asyncGetPostCountList = async (keywordId, cursorId = "") => {
  const fetchInfo = {
    url: `${BASE_URL}/posts/${keywordId}/postCount`,
    params: `?cursorId=${cursorId}`,
  };
```

![image](https://github.com/user-attachments/assets/98362959-1880-4238-814d-397907ebbd44)

Reply:
네네 주간만 진행하기로 한 후에, 해당 코드를 수정하지 못했네요.. 수정하겠습니다!